src/SUMMARY.md

```diff
@@ -3,57 +3,73 @@
   * [Terminology](concepts/terminology.md)
   * [Gaining Access](concepts/gaining_access.md)
   * [Analysis Quick Start](concepts/analysis_intro.md)
+  * [Tools for Data Analysis](tools/interfaces.md)
   * [Intro to Metrics](metrics/intro.md)
   * [Common Analysis Gotchas](concepts/analysis_gotchas.md)
   * [Choosing a Desktop Dataset](concepts/choosing_a_dataset.md)
   * [Choosing a Mobile Dataset](concepts/choosing_a_dataset_mobile.md)
-  * [Glean overview](concepts/glean/glean.md)
-  * [Intro to STMO](tools/stmo.md)
   * [Getting Help](concepts/getting_help.md)
   * [Reporting a problem](concepts/reporting_a_problem.md)
-* [Tools](tools/index.md)
-  * [Project Glossary](tools/projects.md)
-  * [Guiding Principles for Data Infrastructure](tools/guiding_principles.md)
-  * [Overview of Mozilla's Data Pipeline](concepts/pipeline/gcp_data_pipeline.md)
-    * [HTTP Edge Server Specification](concepts/pipeline/http_edge_spec.md)
-    * [Event Pipeline Detail](concepts/pipeline/event_pipeline.md)
-    * [Schemas](concepts/pipeline/schemas.md)
-  * [Analysis Interfaces](tools/interfaces.md)
-  * [SQL Style Guide](concepts/sql_style.md)
 * [Metrics](metrics/index.md)
   * [Definitions](metrics/definitions.md)
     * [Metrics](metrics/metrics.md)
     * [Usage Criteria](metrics/usage.md)
     * [Slicing Dimensions](metrics/dimensions.md)
   * [Metrics Standardization and Policy](metrics/policy.md)
-* [Analysis cookbooks](cookbooks/index.md)
+* [Tutorials & Cookbooks](cookbooks/index.md)
+  * [Analysis cookbooks](cookbooks/index.md)
```

Comment:
This link repeats the top-level cookbooks link. Is that intended? A short blurb about analysis cookbooks would feel more natural for navigating through the content structure.

Reply:
I'll add something. It would be nice if we could autogenerate a table of contents for these types of subsections to make them more useful. /cc @badboy